app/models/report.rb

```diff
@@ -14,6 +14,8 @@ class << self
     include ValueHelper
   end
 
+  COLUMN_SELECTOR_COLUMN_SIZE = 3
```

Comment:
Helps display report columns in neatly lined up columns.

Reply:
another spot where a comment in the code would be useful?

Reply:
Done.